neurom/io/hdf5.py

```diff
@@ -184,29 +184,29 @@ def remove_duplicate_points(points, groups):
         '''
 
         group_initial_ids = groups[:, 0]
+        group_len = len(group_initial_ids)
 
         def _find_last_point(group_id):
             ''' Identifies and returns the id of the last point of a group'''
+            return group_initial_ids[group_id + 1] - 1
```

Comment:
@lidakanari it is actually the ID of the first *point* in the section. BTW, here is the relevant part of the v1 spec (File Format / `structure`):

> **start offset**: The implicit index in the `points` dataset of the first point of the section. _The points in this section go until the next Start offset defined by the next row in the `structure` dataset_, except for the last section in the file, then it runs until the end of the points dataset.

(emphasis mine)

I don't see anything in the v2 spec to deviate from this. But it would be great if more people could check this.